### PR TITLE
feat: Added custom windows / mac installer graphics.

### LINKED ISSUE #3083: feat: Add custom graphics to Electron installers
The current Windows and Mac (.dmg) installers use the default graphics. These graphics should be updated with Composer branding.

### PR DESCRIPTION
**Pull request by tonyanziano**
## Description

This PR adds custom images to the Windows and Mac (.dmg) installers.

## Task Item

Fixes #3083 

### **Before - Windows**

installer-before-1
installer-before-2

### **After - Windows**

installer-after-1
installer-after-2

### **Before - Mac**

installer-mac-before-1

### **After - Mac**

image

### CODE REVIEW

File: Composer/packages/electron-server/electron-builder-config.json [hunk 1]
```diff
@@ -56,6 +57,8 @@
     "runAfterFinish": true,
     "installerIcon": "resources/composerIcon.ico",
     "uninstallerIcon": "resources/composerIcon.ico",
```
Comment (srinaath) [inline Composer/packages/electron-server/electron-builder-config.json:59]: can we remove `resources/` now since we used it above as a directory?

**Comment by srinaath** [comment]
This looks neat!!

**Comment by benbrown** [comment]
I have never been part of a product launch that had an installer that looks as nice as this one!!

File: Composer/packages/electron-server/electron-builder-config.json (hunk 1 above)
Comment (tonyanziano) [reply Composer/packages/electron-server/electron-builder-config.json:59]: good catch! I think we can

**Comment by cwhitten** [comment]
whohoo!

**Pull request merged**